src/components/JsonCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { ExpandLess, ExpandMore } from '@mui/icons-material'
+import {
+  Card,
+  CardActions,
+  CardContent,
+  CircularProgress,
+  IconButton,
+  Typography,
+} from '@mui/material'
+import { ThemeContext } from 'components/UI/Theme/ThemeProvider'
+import { useContext } from 'react'
+import ReactJson from 'react-json-view'
+
+interface IJsonCard {
+  title: string
+  cb: () => void
```

Comment:
`cb: () => void`

Can we make an effort to avoid overly short variable names? In a perfect world, I'd like to know a lot about this function just by looking at its name. Or at least get a general sense of what it's used for...